MANIFEST.in

```diff
@@ -1,2 +1,2 @@
-include LICENSE.txt requirements.txt requirements-grpc.txt pinecone/__version__ pinecone/__environment__
```

Comment:
The requirements files were removed in the poetry migration, I don't think they're needed in the manifest any longer. Let me know if that's incorrect though.